src/modules/profile/components/TabBrands/TabBrands.tsx

```diff
@@ -0,0 +1,57 @@
+import { Grid } from '@material-ui/core';
+import { BrandCard, IBrandCardProps, } from 'modules/common/components/BrandCard';
```

Comment:
I think you need to move BrandCard component from common to the profile module folder